internal/binutils/binutils_test.go

```diff
@@ -220,7 +220,9 @@ func TestDisasm(t *testing.T) {
 	if runtime.GOOS != "linux" && runtime.GOOS != "darwin" {
 		t.Skip("This test only works on Linux or Mac")
 	}
-	testDisasm(t, true)
+	if runtime.GOARCH == "amd64" {
+		testDisasm(t, true)
```

Comment:
It's preferred to skip tests using t.Skip() so that it's clear that it's skipped and why.

I would split this test into two:

```
func TestDisasm(t *testing.T) {
	if runtime.GOOS != "linux" && runtime.GOOS != "darwin" {
		t.Skip("This test only works on Linux or OSX")
	}
	testDisasm(t, false)
}

func TestDisasmIntelSyntax(t *testing.T) {
	if (runtime.GOOS != "linux" && runtime.GOOS != "darwin") || runtime.GOARCH != "amd64" {
		t.Skip("This test only works on x86_64 Linux or x86_64 OSX as it tests Intel asm syntax")
	}
	testDisasm(t, true)
}
```

Reply:
Done.